conda/recipes/dask-cuda/meta.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# Usage:
+#   conda build -c conda-forge -c defaults .
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set cuda_version='.'.join(environ.get('CUDA', 'unknown').split('.')[:2]) %}
+package:
+  name: dask-cuda
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ git_revision_count }}
+  string: cuda{{ cuda_version }}_{{ git_revision_count }}
+  script_env:
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+    - PARALLEL_LEVEL
+    - BUILD_ABI
+
+requirements:
+  build:
+    - python x.x
+    - dask
```

Comment:
@mrocklin I believe this depends on a relatively new version of dask, we should specify that here.

Reply:
Lets go for `dask-core=1.1.4` and drop `dask`